doc/source/tune/faq.rst

```diff
@@ -0,0 +1,145 @@
+.. _tune-faq:
+
+==========================
+Frequently asked questions
+==========================
+
+Here we try to answer questions that come up often. If you still have questions
+after reading this, let us know!
+
+.. contents::
+    :local:
+    :depth: 1
+
+Which search algorithm/scheduler should I choose?
+-------------------------------------------------
+Ray Tune offers :ref:`many different search algorithms <tune-search-alg>`
+and :ref:`schedulers <tune-schedulers>`.
+Deciding on which to use mostly depends on your problem:
+
+* Is it a small or large problem (how long does it take to train? How costly
+  are the resources, like GPUs)? Can you run many trials in parallel?
+* How many hyperparameters would you like to tune?
+* What values are valid for hyperparameters?
+
+**If your model is small**, you can usually try to run many different configurations.
+A **random search** can be used to generate configurations. You can also grid search
+over some values. You should probably still use
+:ref:`ASHA for early termination of bad trials <tune-scheduler-hyperband>`.
+
+**If your model is large**, you can try to either use
+**Bayesian Optimization-based search algorithms** like :ref:`BayesOpt <bayesopt>` or
+:ref:`Dragonfly <Dragonfly>` to get good parameter configurations after few
```

Comment:
Sorry I realise I should probably have commented here rather than Slack but it's too late 😄 https://ray-distributed.slack.com/archives/CNECXMW22/p1597999710018400?thread_ts=1597962894.017600&cid=CNECXMW22